main.bicep

```diff
@@ -350,14 +392,81 @@ resource streamAnalyticsJobs 'Microsoft.StreamAnalytics/streamingjobs@2021-10-01
             }
           }
         }
-      }
-    ]
+      }] : [
+        {
```

Comment:
[discussion] Can we avoid this ternary somehow? If each ASA Bicep record contains details about which bus they should connect to? Not sure if that is possible, as the variable would need to be declared to read `serviceBus` and each queue.

Reply:
The way that I can see is to create seperate resource for anomaly detection task, and leave the old `resource streamAnalyticsJobs` as it was in the beginning. this will eliminate the confusion both in ASA outputs and ASA UDF function

Reply:
Proposed a different approach a bit cleaner